File: PHP/Kode Program/electronic.php
<?php
// Class Electronic mewakili data barang elektronik
class Electronic {
    private $id;         // ID unik barang
    private $name;       // Nama barang
    private $type;       // Jenis barang (TV, Laptop, dll)
    private $price;      // Harga barang
    private $imagePath;  // Path atau URL gambar

    // Constructor
    // Dipanggil saat objek dibuat, langsung isi semua atribut
    public function __construct($id, $name, $type, $price, $imagePath) {
        $this->id = $id;
        $this->name = $name;
        $this->type = $type;
        $this->price = $price;
        $this->imagePath = $imagePath;
    }

    // Getter (ambil data)
    public function getId() { return $this->id; }
    public function getName() { return $this->name; }
    public function getType() { return $this->type; }
    public function getPrice() { return $this->price; }
    public function getImagePath() { return $this->imagePath; }

    // Setter (ubah data)
    public function setName($name) { $this->name = $name; }
    public function setType($type) { $this->type = $type; }
    public function setPrice($price) { $this->price = $price; }
    public function setImagePath($imagePath) { $this->imagePath = $imagePath; }
}
?>

File: python/kode Program/Electronic.py
class Electronic:
    # constructor untuk inisialisasi atribut
    def __init__(self, id: int, name: str, type: str, price: float, image_path: str):
        self._id = id
        self._name = name
        self._type = type
        self._price = price
        self._image_path = image_path

    # getter untuk atribut
    def get_id(self) -> int:
        return self._id

    def get_name(self) -> str:
        return self._name

    def get_type(self) -> str:
        return self._type

    def get_price(self) -> float:
        return self._price

    def get_image_path(self) -> str:
        return self._image_path

    # setter untuk atribut
    def set_name(self, name: str) -> None:
        self._name = name

    def set_type(self, type: str) -> None:
        self._type = type

    def set_price(self, price: float) -> None:
        self._price = price

    def set_image_path(self, image_path: str) -> None:
        self._image_path = image_path